File: 백준/Bronze/2566. 최댓값/최댓값.java
import java.io.*;
import java.util.*;

public class Main {

    static int stoi(String s) {
        return Integer.parseInt(s);
    }

    static int max = Integer.MIN_VALUE, row, col;

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st;

        for (int i = 1; i <= 9; i++) {
            st = new StringTokenizer(br.readLine());
            for (int j = 1; j <= 9; j++) {
                int x = stoi(st.nextToken());
                if (x > max) {
                    max = x;
                    row = i;
                    col = j;
                }
            }
        }

        System.out.println(max);
        System.out.println(row + " " + col);
    }
}

File: 백준/Bronze/2566. 최댓값/최댓값.kt
import java.io.*
import java.util.*

fun main() = with(BufferedReader(InputStreamReader(System.`in`))) {

    var max : Int = Int.MIN_VALUE
    var row : Int = 0
    var col: Int = 0

    for (i in 1..9) {
        val st = StringTokenizer(readLine())
        for (j in 1..9) {
            val x: Int = st.nextToken().toInt()
            if (x > max) {
                max = x
                row = i
                col = j
            }
        }
    }

    println(max)
    println("$row $col")
}